zhmccli/_cmd_partition.py

```diff
@@ -146,6 +145,58 @@ def partition_stop(cmd_ctx, cpc, partition):
     cmd_ctx.execute_cmd(lambda: cmd_partition_stop(cmd_ctx, cpc, partition))
 
 
+@partition_group.command('dump', options_metavar=COMMAND_OPTIONS_METAVAR)
+@click.argument('CPC', type=str, metavar='CPC')
+@click.argument('PARTITION', type=str, metavar='PARTITION')
+@click.option('--volume', type=str, required=True,
+              help='The storage volume that contains the dump program. '
+              'For CPCs with the storage management feature (z14 and later): '
+              'A string of the form "SG.SV" where SG is the name '
+              'of the storage group resource attached to the partition and SV '
+              'is the name of the storage volume in that storage group, or of '
+              'the form "UUID" where UUID is the UUID of the storage volume '
+              'on the storage array (also shown in the HMC GUI). The storage '
+              'volume may be of type FCP or FICON. '
+              'For CPCs without the storage management feature (z13 and '
+              'earlier): A string of the form "HBA.WWPN.LUN", where HBA is the '
+              'name of the HBA resource in the partition and WWPN and LUN '
+              'identify the storage array and storage volume thereon. The '
+              'storage volume must be of type FCP.')
+@click.option('--lba', type=str, required=False, default='0',
+              help='The logical block number of the anchor point for locating '
+              'the dump program on the storage volume. Default: 0')
```

Comment:
General question: Why do we duplicate the default values? Why do we not use the option `show_default=True` instead https://click.palletsprojects.com/en/7.x/api/#click.Option (maybe we can change this in a separate patch)?

Reply:
This would definitely be a separate issue/PR because it would affect all commands. I have opened issue #125.
RESOLVED